### ISSUE TITLE
strawR update

### ISSUE THREAD

**Issue by sa501428** [aidenlab/straw#107]
The C++ and python flavors for straw have been pretty significantly improved/upgraded. It would be great if some of these improvements could also get pulled into the strawR version and CRAN could also be updated.

**Comment by moshe-olshansky** [comment]
Isn't strawR a wrapper to C++?

**Comment by sa501428** [comment]
Yes. So essentially the C++ scripts in the R folder need to get updated.
But we've also added persistent data structures, so those may be good to
add as well if possible.

**Comment by cwenger** [comment]
@sa501428 Yes, I'll try to do this at some point!
